### PR TITLE
018 pawn

### PR DESCRIPTION
**Pull request by Gonaline**

### CODE REVIEW

**Review by FrancoisSalmon — DISMISSED** [review]
ok for me!

**Review by FrancoisSalmon — DISMISSED** [review]
ok for me

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/src/components/Question/index.jsx [hunk 1]
```diff
@@ -1,15 +1,27 @@
 import PropTypes from "prop-types";
 import Style from "./style";
 
-export default function Question({ data: { question, category } }) {
+export default function Question({
+  data: { question },
+  categoryName,
+  categoryColor,
```
Comment (LoicBrassart) [inline frontend/src/components/Question/index.jsx]: Are these informations not stored in the Context ? Passing them as props here is useless :(

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Now that you have the Context set in place, use it instead of props for data already in it

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**